plugins/PostProcessingPlugin/PostProcessingPlugin.py

```diff
@@ -156,6 +156,24 @@ def loadAllScripts(self) -> None:
     #   This should probably only be done on init.
     #   \param path Path to check for scripts.
     def loadScripts(self, path: str) -> None:
+
+        if ApplicationMetadata.IsEnterpriseVersion:
+            # Delete all __pycache__ not in installation folder, as it presents a security risk.
+            # Also it prevents this very strange scenario:
+            #  - Copy an existing script from the postprocessing-script folder to the appdata scripts folder.
+            #  - Also copy the entire __pycache__ folder from the first to the last location.
+            #  - Leave the __pycache__ as is, but write maliscous code just before the class begins.
```

Comment:
`malicious`

Reply:
There are some other things with those comments as well. I'll fix it.

Reply:
commit aaee65136